vms/platformvm/vm_test.go

```diff
@@ -524,16 +523,6 @@ func TestGenesis(t *testing.T) {
 		}
 	}
 
-	// Ensure current validator set of primary network is correct
-	vdrSet, ok := vm.Validators.GetValidators()
-	if !ok {
-		t.Fatalf("Missing the primary network validator set")
-	}
-	currentValidators := vdrSet.List()
-	if len(currentValidators) != 0 {
-		t.Fatal("vm's current validator set is wrong")
-	}
-
```

Comment:
Why is this removed? Currently this wouldn't pass? It seems pretty basic though, that `vm.Validators.GetValidators` should return a non-empty list of validators?